apps/nextjs-website/src/components/organisms/GuideContent/components/Hint.tsx

```diff
@@ -0,0 +1,16 @@
+import Alert, { AlertColor } from '@mui/material/Alert';
+import { ReactNode } from 'react';
+
+export type HintProps = {
```

Comment:
Are you going to keep this type or would you like to keep the same type under `packages/gitbook-docs/src/markdoc/schema/hint.ts`?

Reply:
We are going to use the `packages/gitbook-docs/src/markdoc/schema/hint.ts` Props. I'mh going to migrate all of them before undraft this PR